sormas-backend/src/main/java/de/symeda/sormas/backend/clinicalcourse/ClinicalVisitFacadeEjb.java

```diff
@@ -226,17 +226,22 @@ public ClinicalVisitDto getClinicalVisitByUuid(String uuid) {
 		UserRight._CLINICAL_VISIT_CREATE,
 		UserRight._CLINICAL_VISIT_EDIT })
 	public ClinicalVisitDto saveClinicalVisit(ClinicalVisitDto clinicalVisit, String caseUuid) {
-		return saveClinicalVisit(clinicalVisit, caseUuid, true);
+		return saveClinicalVisit(clinicalVisit, caseUuid, true, false);
 	}
 
-	@RightsAllowed({
-		UserRight._CLINICAL_VISIT_CREATE,
-		UserRight._CLINICAL_VISIT_EDIT })
-	public ClinicalVisitDto saveClinicalVisit(ClinicalVisitDto clinicalVisit, String caseUuid, boolean handleChanges) {
+	@RightsAllowed(UserRight._CASE_CREATE)
+	public ClinicalVisitDto saveMergedCasesClinicalVisit(ClinicalVisitDto clinicalVisit, String caseUuid, boolean handleChanges) {
```

Comment:
```suggestion
	public ClinicalVisitDto saveClinicalVisitForMergedCase(ClinicalVisitDto clinicalVisit, String caseUuid, boolean handleChanges) {
```

Bit more self-explanatory to me

Reply:
updated the method name